crm_properties.go

```diff
@@ -0,0 +1,107 @@
+package hubspot
+
+import (
+	"fmt"
+	"time"
+)
+
+const (
+	crmPropertiesPath = "properties"
+)
+
+type CrmPropertiesList struct {
+	Results []CrmProperty
+}
+
+type CrmProperty struct {
+	Calculated       bool   `json:"calculated"`
+	Description      string `json:"description"`
+	DisplayOrder     int    `json:"displayOrder"`
+	ExternalOptions  bool   `json:"externalOptions"`
+	FieldType        string `json:"fieldType"`
+	FormField        bool   `json:"formField"`
+	GroupName        string `json:"groupName"`
+	HasUniqueValue   bool   `json:"hasUniqueValue"`
+	Hidden           bool   `json:"hidden"`
+	HubspotDefined   bool   `json:"hubspotDefined"`
+	Label            string `json:"label"`
+	ModificationMeta CrmPropertyModificationMeta
+	Name             string `json:"name"`
+	Options          []CrmPropertyOptions
+	Type             string `json:"type"`
+	CreatedUserId    string
+	UpdatedUserId    string
+	CreatedAt        *time.Time
+	UpdatedAt        *time.Time
+}
+
+type CrmPropertyModificationMeta struct {
+	Archivable       bool `json:"archivable"`
+	ReadOnlyDefition bool `json:"readOnlyDefinition"`
+	ReadOnlyValue    bool `json:"readOnlyValue"`
+}
+type CrmPropertyOptions struct {
+	DisplayOrder int    `json:"displayOrder"`
+	Hidden       bool   `json:"hidden"`
+	Label        string `json:"label"`
+	Value        string `json:"value"`
+}
+
+// CrmPropertiesService is an interface of CRM properties endpoints of the HubSpot API.
+// Reference: https://developers.hubspot.com/docs/api/crm/properties
+type CrmPropertiesService interface {
+	List(string) (*CrmPropertiesList, error)
+	Create(string, interface{}) (*CrmProperty, error)
```

Comment:
I would like to see parameter names added.
It is fine without it by convention, but it is difficult to know which is used for what, such as (string, string), etc., so we would like to specify it clearly.